### PR TITLE
Fix template

### PR DESCRIPTION
**Pull request by Erin-Mounts**
so it doesn't append the same relationship items to the existing list every time it updates an object

### CODE REVIEW

File: BridgeSDK/BridgeAPI/SBBActivityManager.m [hunk 1]
```diff
@@ -138,7 +140,8 @@ - (NSURLSessionDataTask *)getScheduledActivitiesForDaysAhead:(NSInteger)daysAhea
         savedTasks = [self filterTasks:tasks.items forDaysAhead:kMaxAdvance andDaysBehind:kDaysToCache excludeStillValid:YES];
     }
 
-    return [self.networkManager get:kSBBActivityAPI headers:headers parameters:@{@"daysAhead": @(daysAhead), @"offset": [[NSDate date] ISO8601OffsetString]} completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
+    // always request the maximum days ahead from the server so we have them cached
+    return [self.networkManager get:kSBBActivityAPI headers:headers parameters:@{@"daysAhead": @(kMaxAdvance), @"offset": [[NSDate date] ISO8601OffsetString]} completion:^(NSURLSessionDataTask *task, id responseObject, NSError *error) {
```
Comment (syoung-smallwisdom) [inline BridgeSDK/BridgeAPI/SBBActivityManager.m]: How will caching work for when the schedules change (b/c of changes to data groups)? This still does a replacement, right? (i.e. server wins)

Comment (Erin-Mounts) [inline BridgeSDK/BridgeAPI/SBBActivityManager.m]: server "wins", but we cheat on this one: before updating, SBBActivityManager saves everything we want to keep that the server would leave out/overwrite/delete (i.e. everything going back 1 week in the past and everything from today on that's been marked finished), and then adds it back in after updating from server.

Comment (syoung-smallwisdom) [inline BridgeSDK/BridgeAPI/SBBActivityManager.m]: cool, thanks

**Comment by syoung-smallwisdom** [comment]
looks good

**Pull request merged**